doc/docs/1.12.x/pachhub/pachhub_getting_started.md

```diff
@@ -1,181 +1,101 @@
-Pachyderm contexts, see [Connect by using a Pachyderm Context](../../deploy-manage/deploy/connect-to-cluster/#connect-by-using-a-pachyderm-context).
-
-To set the correct Pachyderm context, you need to use the hostname
-of your cluster that is available in the Pachyderm Hub UI under **Connect**.
+After your cluster creation, open a terminal window and [install 'pachctl'](https://docs.pachyderm.com/latest/getting_started/local_installation/#install-pachctl).
 
-!!! note
+!!! Warning
     `kubectl` commands are not supported for the clusters deployed
     on Pachyderm Hub.
+## 3-4 Configure your Pachyderm context and login to your cluster by using a one-time authentication token
+1. To configure a Pachyderm context and log in to your cluster
+(i.e. have your`pachctl` point to your new cluster), click the **Connect** link on your cluster name in the Pachyderm Hub UI.
 
-To connect to your cluster, complete the following steps:
+      ![Pachyderm Cluster running](../images/pachhub_cluster_running.png)
 
-1. On your local computer, open a terminal window.
-1. Install or upgrade  `pachctl` as described in
-   [Install pachctl](../../getting_started/local_installation/#install-pachctl).
+      In your terminal window, copy, paste, and run the commands 1,2,3 listed in the instructions.
 
-1. Verify your `pachctl` version:
+      ![Pachyderm Cluster connect](../images/pachhub_cluster_connect.png)
 
-   ```shell
-   pachctl version --client-only
-   ```
+      **Note**
+            Your token has a limited lifetime. The following error means that your authentication
+            token has expired. Close the pop-up window and re-connect.
+            ```shell
+               error authenticating with Pachyderm cluster: otp is invalid or has expired
+            ```
 
-   **System Response:**
-
-   ```shell
-   {{ config.pach_latest_version }}
-   ```
-
-1. Configure a Pachyderm context and log in to your
-   cluster by using a one-time authentication token:
-
-   1. In the Pachyderm Hub UI, click **Connect** next to your cluster.
-   1. In your terminal window, copy, paste, and run the commands listed in
-      the instructions.
-      These commands create a new Pachyderm context with your cluster
-      details on your machine.
-
-      **Note:** If you get the following error, that means that your authentication
-      token has expired:
-
-      ```shell
-      error authenticating with Pachyderm cluster: /pachyderm_auth/auth-codes/ e14ccfafb35d4768f4a73b2dc9238b365492b88e98b76929d82ef0c6079e0027 not found
-      ```
-
-      To get a new token, refresh the page. Then, use
-      the new token to authenticate.
-
-   1. Verify that you have set the correct context:
+1. Verify that you have set the correct context:
 
       ```shell
       pachctl config get active-context
       ```
-
-      **System Response:**
-
-      ```shell
-      test-svet-cc0mi51i52
+      **System response**
+      ```
+      jolly-caribou
```

Comment:
It might be valuable to note that jolly-caribou is whatever you typed into the "create workspace" dialog.  (Maybe that's clear from the screenshots though.)